src/librustc/ty/print/mod.rs

```diff
@@ -7,6 +7,7 @@ use rustc_data_structures::fx::FxHashSet;
 
 // `pretty` is a separate module only for organization.
 mod pretty;
+pub mod obsolete;
 pub use self::pretty::*;
```

Comment:
Hmm, maybe put `mod obsolete; ` below the `pretty` reexport (with an empty line before it).